src/modules/px4iofirmware/protocol.h

```diff
@@ -240,6 +240,8 @@ enum {							/* DSM bind states */
 
 #define PX4IO_P_SETUP_AIRMODE 			27 	/* air-mode */
 
+#define PX4IO_P_SETUP_ENABLE_FLIGHTTERMINATION	28	/* flight termination circuit breaker */
```

Comment:
I would not use the name circuit breaker here but just leave it at what it is: "Flight termination enabled".
The circuit breaker stuff is just confusing.